.github/workflows/pr-test.yml

```diff
@@ -235,7 +235,7 @@ jobs:
         run: |
           pip install --upgrade pip
           pip install -e "python[all]"
-          pip install transformers==4.44
+          pip install transformers==4.45.2
```

Comment:
Can you make this a separate PR? This will also make you a contributor and it will make your future commits trigger CI automatically

Reply:
Let me raise another PR to upgrade these dependencies.

Reply:
sounds good